ui/components/ui/review-spending-cap/review-spending-cap.js

```diff
@@ -92,7 +93,11 @@ export default function ReviewSpendingCap({
                 <i className="fa fa-exclamation-triangle review-spending-cap__heading-title__tooltip__warning-icon" />
               )}
               {Number(tokenValue) === 0 && (
-                <i className="far fa-question-circle review-spending-cap__heading-title__tooltip__question-icon" />
+                <Icon
+                  className="review-spending-cap__heading-title__tooltip__question-icon"
+                  name={ICON_NAMES.QUESTION}
+                  color={Color.iconDefault}
+                />
               )}
             </Tooltip>
           </Box>
```

Comment:
Adding screenshot for other reviewers

<img width="728" alt="Screenshot 2023-02-21 at 10 08 28 PM" src="https://user-images.githubusercontent.com/8112138/220527752-7719a956-3204-48ce-920b-7bb21700d491.png">